sympy/core/assumptions.py

```diff
@@ -274,8 +291,12 @@
     'infinite       ==  !finite',
     'noninteger     ==  extended_real & !integer',
     'extended_nonzero == extended_real & !zero',
-])
+    ])
+    return _assume_rules
+
 
+#_assume_rules = _load_pre_generated_assumption_rules()
+_assume_rules =_generate_assumption_rules()
```

Comment:
Looks like the main intention of this PR was not really achieved because the wrong line is commented out here.